test/index.js

```diff
@@ -348,7 +351,7 @@ test('combine throws an error when passed a module with a create but no gives',
 test('combine throws an error when passed a module with a create when gives a string', function (t) {
   const a = {
     gives: 'nope',
-    create: (api) => {} 
+    create: (api) => {}
```

Comment:
does this work? Should this be `create: (api) => () => {}`
as create should return a function